### PR TITLE
Fix for #278 Bug: Variables and ElementInstances not stored in DB

### PR DESCRIPTION
**Pull request by nitram509**
The root cause was a broken check for duplicates in the DB.
This solution favors creating new ID unique index fields for the two mentioned tables,
in order to feed all data (variables and element_instances) into the DB.
Assuming, there are no duplicate events exported, there's no need for a duplicate check.

Additionally, two indices are added, for performance reasons.

Unfortunately, because JPA requires an @id field, a new field was introduced
and there's an UPGRADE instruction documented, so people out there would be able to migrate their installations.
In other words, this PR contains a breaking change in DB structures.

Also, if you look at what files are changed, this PR contains two commits.
The first one is pure refactoring, in order to split the ZeebeImportService class into multiple importer-classes.
This is mainly for readability. I hope you find this useful.
The second commit is the bug fix itself.

Any feedback is welcome.

### CODE REVIEW

**Comment by saig0** [comment]
@nitram509 thank you for your contribution :tada:

Please have a look at my comment: https://github.com/camunda-community-hub/zeebe-simple-monitor/issues/278#issuecomment-894200353

Other comments to the PR,
* splitting into different classes is okay for me
* adding the indexes is a good idea but I would prefer a separate PR 
* I appreciate the update guide :+1:

**Comment by nitram509** [comment]
I will close this PR and create separate ones, as recommended.

**Comment by nitram509** [comment]
See #284 for part 1, the refactoring.
